lemongrab/templates/index.html

```diff
@@ -22,20 +25,28 @@
       <h3>Add a New Task</h3>
       <form id="new-task-form">
         <input type="text" placeholder="Task" />
-        <input type="button" class="button" value="Add Task" onclick="TODO.addTask();" />
+        <input type="button" class="button" value="Add Task"
+          onclick="TODO.addTask();" />
       </form>
     </div>
 
-    <div id="task-lists">
-      <div class="task-area" id="incomplete">
-        <h3>To Do</h3>
-      </div>
+    <div class="task-area" id="task-list">
+      <h3>To Do</h3>
+    </div>
 
-      <div class="task-area" id="complete">
-        <h3>Completed</h3>
+    <!-- Template for tasks to display in the list above -->
+    <script id="task-template" type="text/x-handlebars-template">
+      <div class="task" id="task-{{id}}">
+        <div class="task-body">{{description}}</div>
+        <div class="task-incomplete">Not completed</div>
+        <div class="task-control">
+          <input type="button" id="toggle" value="Complete"
```

Comment:
Good catch.

In HTML5 the closing '/' is depracated and shouldn't be used for void elemets. It's supported in all browsers and won't cause a problem, but its really an ignored syntax error. So this instance is correct and the others are wrong. Here's google's style guide's statement on the matter: http://google-styleguide.googlecode.com/svn/trunk/htmlcssguide.xml?showone=Document_Type#Document_Type

I'll fix them.